.github/workflows/preview-site.yml

```
@@ -15,10 +15,10 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@main
 
-      - name: Setup Node.js 14.x
+      - name: Setup Node.js 16.x
         uses: actions/setup-node@main
         with:
-          node-version: 14.x
+          node-version: 16.x
```

Forgot to add the comment before approving
Now that there's a volta config you could replace this with:
```suggestion
          node-version-file: 'package.json'